libraries/Adafruit_LittleFS/src/Adafruit_LittleFS_File.h

```diff
@@ -69,13 +75,21 @@ class File : public Stream
     uint32_t size (void);
 
     void close (void);
+
+    __attribute__((deprecated("Recommend use of isOpen() for clarity")))
```

Comment:
it is how Arduino design and used to test the file through out its examples. Don't be bothered with it ;)

Reply:
Ok...